package.json

```diff
@@ -38,10 +38,10 @@
     "marked": "^0.3.2",
     "mocha": "^2.5.3",
     "mocha-loader": "^1.0.0",
-    "mocha-webpack": "^0.4.0",
+    "mocha-webpack": "next",
```

Comment:
I'd rather not pin to next since that will change in the future and might lead to an issue.

Did you have a problem setting this directly to v1.0.0-rc.1?